src/components/WalletConnect/CreateTokenRequest.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  StyleSheet,
+  View,
+  ScrollView,
+  TouchableWithoutFeedback,
+  Text,
+  Image,
+} from 'react-native';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigation } from '@react-navigation/native';
+import { numberUtils } from '@hathor/wallet-lib';
+import { t } from 'ttag';
+import {
+  createTokenRetry,
+  createTokenRetryDismiss,
+  setCreateTokenStatusReady,
+  walletConnectAccept,
+  walletConnectReject
+} from '../../actions';
+import { COLORS } from '../../styles/themes';
+import NewHathorButton from '../NewHathorButton';
+import { DappContainer } from './NanoContract/DappContainer';
+import { commonStyles } from './theme';
+import { FeedbackContent } from '../FeedbackContent';
+import { DEFAULT_TOKEN, WALLETCONNECT_CREATE_TOKEN_STATUS } from '../../constants';
+import FeedbackModal from '../FeedbackModal';
+import Spinner from '../Spinner';
+import errorIcon from '../../assets/images/icErrorBig.png';
+import checkIcon from '../../assets/images/icCheckBig.png';
+
+const condRenderData = (
+  attribute,
+  title,
+  separator,
+  formatter = (val) => val,
+) => {
+  if (attribute != null) {
+    return (
+      <>
+        { separator && <View style={commonStyles.cardSeparator} /> }
+        <View>
+          <Text style={styles.property}>{ title }</Text>
+          <Text style={[styles.value, styles.bold]}>
+            { formatter(attribute) }
+          </Text>
+        </View>
+      </>
+    );
+  }
+
+  return null;
+};
+
+function renderBoolean(bool) {
```

Comment:
[suggestion] You could call it `renderBooleanFormatter` to add some semantic to it. Also, you can say in a docstring that this component should be used as a parameter of `condRenderData`.

Reply:
Refactored, thanks